src/test/java/integration/ActionDoIT.java

```diff
@@ -53,14 +53,14 @@ public void badLocatorTest() {
         finish();
     }
 
-    @Test(groups = {"integration", "action", "do", "url"},
+    @Test(groups = {"integration", "action", "do", "url", "browser"},
             description = "An integration test to check the goToURL method")
     public void goToURLTest() {
         // use this object to manipulate the app
         App app = this.apps.get();
         // perform some actions
-        app.goToURL("https://www.google.com/");
-        app.azzert().urlEquals("https://www.google.com/");
+        app.goToURL("https://www.bing.com/");
```

Comment:
lol

Reply:
it's an html unit issue...i'd rather not, but google made a change, and I'd rather not remove the test